testing/experiment.py

```diff
@@ -200,6 +220,10 @@ def run(self):
 
         with open(f"{self.output_dir}/experiment_results.json", "w") as json_file:
             json.dump(results, json_file)
+        with open(
```

Comment:
files are small so maybe use pretty print? To have a look on the results without plots